aws/rds.md

```diff
@@ -0,0 +1,26 @@
+# AWS Relational Database Service (RDS)
+
+This doc presents guides on working with AWS RDS, Amazon's Database-as-a-Service provider.
+
+## Connecting with an EC2 instance
+
+In order to connect to an RDS instance from an EC2 instance, you'll need to configure the security rules for the database to allow access from the instance. Broadly speaking, AWS allows you to do this by 1) creating the database and the EC2 instance in the same Virtual Private Cloud (VPC) and 2) creating an inbound rule for the database's security group that permits access to PostgreSQL from the EC2 instance's security group.
+
+These instructions are loosely based on [AWS's official documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_VPC.Scenarios.html#USER_VPC.Scenario1).
+
+- Navigate to the page for your database in the RDS console.
+- Under `Connectivity & security > Networking > VPC`, copy the ID of the VPC that the database runs in.
+- Navigate to the EC2 console and begin to launch an EC2 instance [following our guide](https://github.com/datamade/deploy-a-site/blob/master/Launch-a-new-EC2-Instance.md)
+- During Step 3, the "Configure Instance Details" page, specify the following options:
```

Comment:
formatting note: can you nest the "during step..." items under the preceding point, since they're variations on the guide?

Reply:
I like that! Done.